lib/compat/wordpress-6.1/class-gutenberg-rest-navigation-controller.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Templates_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Base Templates REST API Controller.
+ */
+class Gutenberg_REST_Navigation_Controller extends WP_REST_Posts_Controller {
+
+	/**
+	 * Registers the controllers routes.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+
+		parent::register_routes();
+
+		// Lists a single nav item based on the given id.
```

Comment:
```suggestion
		// Lists a single nav item based on the given slug.
```

These routes are registered purely for handling `slugs`.